<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRun_BusWindow</class>
 <widget class="QDialog" name="NewRun_BusWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новий рейс : автобус</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" alignment="Qt::AlignHCenter">
    <widget class="QDialogButtonBox" name="newrun_bussubmit_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Виберіть серед доступних автобусів</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="newrun_Buses_general_tableWidget">
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>№</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Місткість</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Тип тр. засобу</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Початковий пробіг</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Споживання палива</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Останій ТО</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTableWidget" name="newrun_Bus_runs_registered_tableWidget">
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>№</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Відправлення</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Прибуття</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Час відправлення</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Час прибуття</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="newrun_bus_runs_registered_hide_pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Сховати</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
